apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/QueryOptions.cs

```diff
@@ -100,6 +100,15 @@ public sealed class QueryOptions : JobCreationOptions
         /// </summary>
         public bool? DryRun { get; set; }
 
+        /// <summary>
+        /// If set, determines whether parameters are named or positional. If not set,
+        /// this will default to named parameters. When named parameters are used, all parameters
+        /// must have a name. When positional parameters are used, parameters may still have names,
+        /// but don't have to.
+        /// </summary>
+        /// <remarks>This property is not used by the legacy query overloads using <see cref="BigQueryCommand"/>.</remarks>
```

Comment:
Put in a TODO to remove this remark?

Reply:
No need - I've deliberately made it refer to BigQueryCommand so that we'll get an error when we delete that, at which point I'll remove the remark.